app/views/layouts/_font_loading.html.haml

```diff
@@ -0,0 +1,16 @@
+/ We want to get the holding font rendered straight away
+%link{ rel: "preload", href: asset_path('poppins-v20-latin-regular.woff2'), as: "font", type: "font/woff2", crossorigin: :anonymous }
+:css
+  @font-face {
+    font-display: fallback;
+    font-family: PoppinsInitial;
+    src: url(#{asset_path('poppins-v20-latin-regular.woff2')}) format('woff2');
+  }
+  body {
+    --body-font: Poppins, PoppinsInitial, serif;
+    font-family: var(--body-font);
+    -webkit-font-smoothing: antialiased;
+  }
+  body.fonts-loaded {
+    --body-font: Poppins, serif;
+  }
```

Comment:
What does this do?

Reply:
@iHiD Something for you to answer when you're back

Reply:
It's not used any more. This is the route taken in the article I linked to, but my way is better and doesn't require JavaScript. I'll simplify this now.